index.bs

```diff
@@ -424,14 +427,18 @@ Some {{XRSessionMode}}s request certain [=feature names=] as optional features b
     </tr>
   </thead>
   <tbody>
+    <tr>
+      <td>{{XRReferenceSpaceType/"viewer"}}</td>
+      <td>{{XRSessionMode/"inline"}}, {{XRSessionMode/"immersive-vr"}}</td>
```

Comment:
I've generally been including "immersive-ar" everywhere until we do the module split.  Mostly because I want to make sure I don't miss anything at the time that will need to be called out in the new documents.